cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -116,8 +116,11 @@ public void execute() {
                     srcInput = scanner.nextLine().trim();
 
                     if (srcInput.equalsIgnoreCase("service") || srcInput.equalsIgnoreCase("s") || srcInput.isEmpty()) {
-                        out.print("Package for the service : (no package) ");
-                        String packageName = scanner.nextLine().trim();
+                        String packageName;
+                        do {
+                            out.print("Package for the service : (no package) ");
```

Comment:
Shall we get rid of the space before the colon ?

Reply:
Fixed with 45ff1fd